src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/Cohost/CohostWorkspaceSpellCheckEndpoint.cs

```diff
@@ -17,12 +16,9 @@ namespace Microsoft.VisualStudio.Razor.LanguageClient.Cohost;
 [ExportCohostStatelessLspService(typeof(CohostWorkspaceSpellCheckEndpoint))]
 [method: ImportingConstructor]
 #pragma warning restore RS0030 // Do not use banned APIs
-internal sealed class CohostWorkspaceSpellCheckEndpoint(
-    IRemoteServiceInvoker remoteServiceInvoker)
+internal sealed class CohostWorkspaceSpellCheckEndpoint()
     : AbstractRazorCohostRequestHandler<VSInternalWorkspaceSpellCheckableParams, VSInternalWorkspaceSpellCheckableReport[]>
```

Comment:
nit: The don't do any harm, but I think you can remove the `[method: ImportingConstructor]` and the empty parentheses. Also, this seems OK to be a single line now.